expyfun/_experiment_controller.py

```diff
@@ -933,6 +935,8 @@ def get_presses(self, live_keys=None, timestamp=True, relative_to=None):
             A time relative to which timestamping is done. Ignored if
             timestamp==False.  If ``None``, timestamps are relative to the time
             `listen_presses` was last called.
+        releases : False | bool
+            Whether key-release events should be returned as well.
```

Comment:
@Eric89GXL just convinced me that rather than `releases=False` (or `True`) we have a `type` parameter that takes a string value (`presses`, `releases`, or `both`).